packaging_automation/upload_to_package_cloud.py

```diff
@@ -116,15 +115,15 @@ def package_exists(package_cloud_token: str, repo_owner: str, repo_name: str, pa
     parser.add_argument('--package_cloud_api_token', required=True)
     parser.add_argument('--repository_name', required=True, choices=supported_repos)
     parser.add_argument('--output_file_path', required=True)
-    parser.add_argument('--exec_folder_path', required=True)
+    parser.add_argument('--current_branch', required=True)
```

Comment:
Now we have --current_branch param but it is not set via github actions job or something, is it what we want ?

Reply:
it will be set on Github Actions pipeline. I will send the pipeline code in another pipeline in packaging repo